docs/core/about.md

```diff
@@ -5,47 +5,45 @@ ms.date: 09/17/2019
 ---
 # About .NET Core
 
-.NET Core has the following characteristics:
+.NET Core is/has:
 
 - **Cross-platform:** Runs on Windows, macOS, and Linux [operating systems](https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md).
-- **Consistent across architectures:** Runs your code with the same behavior on multiple architectures, including x64, x86, and ARM.
-- **Command-line tools:**  Includes easy-to-use command-line tools that can be used for local development and in continuous-integration scenarios.
-- **Flexible deployment:** Can be included in your app or installed side-by-side (user-wide or system-wide installations). Can be used with [Docker containers](docker/introduction.md).
-- **Compatible:** .NET Core is compatible with .NET Framework, Xamarin, and Mono, via [.NET Standard](../standard/net-standard.md).
-- **Open source:** The .NET Core platform is open source, using MIT and Apache 2 licenses. .NET Core is a [.NET Foundation](https://dotnetfoundation.org/) project.
-- **Supported by Microsoft:** .NET Core is supported by Microsoft, per [.NET Core Support](https://dotnet.microsoft.com/platform/support/policy).
+- **Open source:** The .NET Core platform is [open source](https://github.com/dotnet/core), using MIT and Apache 2 licenses. .NET Core is a [.NET Foundation](https://dotnetfoundation.org/) project.
+- **Modern:** It implements modern paradigms like asyncronous programming, no-copy patterns using structs, and resource governance for containers.
+- **Performance:**  Delivers [high performance](https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-core-3-0/) with features like [hardware intrinsics](https://devblogs.microsoft.com/dotnet/hardware-intrinsics-in-net-core/), [tiered compilation](https://github.com/dotnet/coreclr/blob/master/Documentation/design-docs/tiered-compilation.md), and [Span<T>](../standard/memory-and-spans/index.md).
+- **Consistent across environments:** Runs your code with the same behavior on multiple OSes and architectures, including x64, x86, and ARM.
```

Comment:
The acronym for operating systems is actually OSs which can be usually confused with OSS. I'd just spell out.
https://itknowledgeexchange.techtarget.com/writing-for-business/the-plural-of-os/

```suggestion
- **Consistent across environments:** Runs your code with the same behavior on multiple operating systems and architectures, including x64, x86, and ARM.
```